src/components/ActionButton/ActionButton.tsx

```diff
@@ -0,0 +1,34 @@
+import { Button, Flex } from '@patternfly/react-core';
```

Comment:
This file is named like `ActionButton.tsx` and it exports two components:
1) ActionButton
2) ActionHeader

Which one is considered to be the default one that should be commonly used? Maybe creating new file for each component would be better? Or removing `export` from `ActionButton` and renaming this file to `ActionHeader.tsx`? Let's discuss it.

Reply:
ActionButton is only the main button component whereas the action header uses the button to hide some more logic (so it doesnt have to repeat). I think both may be used and it may be difficult to decide which is the primary one. I will create a new file because these components may get more complex in the future.